src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomColorIndex = new Random().nextInt(Color.values().length);
+        Color randomColor = Color.values()[randomColorIndex];
+        return randomColor.toString();
```

Comment:
[Use name() for getting String representation of enum constants](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)